.changeset/cool-scissors-invite.md

```diff
@@ -0,0 +1,17 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Add the option to define custom header title and/or subtitle for the catalog layout page that would replace the greetings in random languages.
+
+```js
+<Route
+  path="/catalog"
+  element={
+    <CatalogIndexPage
+      customHeaderTitle="MyCompany Service Catalog"
+      customHeaderSubtitle="Some subtitle text"
```

Comment:
To be honest I don't think we want to set this precedent right now, as it will move across to other plugins and it's likely not what we want to be doing in the end. My thought right now is that we want to move the header out of the plugin altogether and make that part of an app-defined layout component, as it's too much burden on the plugin to have to always be providing configuration points for the header. Basically the plugin should be able to provide extra information for the header, not the other way around.